src/Raven.Server/Documents/PeriodicBackup/Azure/RavenAzureClient.cs

```diff
@@ -33,12 +33,13 @@ public class RavenAzureClient : RavenStorageClient
         private readonly byte[] _accountKey;
         private readonly string _containerName;
         private readonly string _serverUrlForContainer;
-        private const string AzureStorageVersion = "2017-04-17";
+        private const string AzureStorageVersion = "2019-02-02";
```

Comment:
Why this was changed? Can this cause some issues now? What are the benefits?

Reply:
This is the version that supports bulk instert